### DISCUSSION TITLE
Welcome to RetroAssembly Discussions!

### DISCUSSION THREAD

**Discussion by arianrhodsandlot** [arianrhodsandlot/retroassembly#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SirBossJam** [comment]
I just wanted to say how great this project is. I haven't been this excited for something around emulation in a long time. After 15 years or so of configuring emulation setups, it has really become a chore and I just haven't been into it much lately because of that. But now, having my library and saves synced and playable on whatever device I happen to be on is super convenient, and I've been having a lot of fun with RetroAssembly these past few weeks. I know it will be a while before emulation in a browser is on par with local installs, but wow, this is really impressive! Support for my favorite systems are already here, so I'm good with that. I only wish good things for this project, and hopefully I can help along the way. Thank you for all your hard work.

**Reply by arianrhodsandlot** [reply]
Your affirmation encouraged me a lot. Enjoy playing and feel free to share this project to others who are interested in retro gaming :)
